pyproject.toml

```diff
@@ -1,8 +1,14 @@
-[tool.poetry]
+[project]
 name = "tom-alertstreams"
```

Comment:
I'm open to changing the name of the repo to `tom_alertstreams` to be more consistent with the rest of our repos.

Reply:
This is the pypi name that has to be like this.
We can re-name the repo...